composer.json

```diff
@@ -5,17 +5,23 @@
     "homepage": "http://cakephp.org",
     "license": "MIT",
     "require": {
+        "php": "^8.0",
```

Comment:
The CI seems uses minimal php 8.0, so I added php requirement here.

Reply:
Hi @samsonasik I think it would be a problem for PHP7.4 users. 

CakePHP 3.x has PHP requirement like this  ` "php": ">=5.6.0,<8.0.0",` Where as CakePHP 4.x       `  "php": ">=7.4.0",
`

Reply:
This repo CI is using php 8.0 for minimal php version

https://github.com/cakephp/upgrade/blob/9823a2af5c4f3d96ae3bbdad58e4a6fddaafcd69/.github/workflows/ci.yml#L17

so I downgraded the rules to use php 8.0 syntax.